src/net/Lib/Images/DefaultImageProvider.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Net.Http.Json;
+using Lib.Model;
+
+namespace Lib.Images
+{
+    public class DefaultImageSource : IImageProvider
```

Comment:
Since this only works with the specific schema returned by this API, should we call this RedditMemeProvider